Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'sneakers', '~> 2.11'
 gem 'solrizer'
 
 # DLSS gems
-gem 'dor_indexing', '~> 1.1'
+gem 'dor_indexing', '~> 1.1', '>= 1.2.0'
```

Comment:
not a blocker, but i think you could just do `'~> 1.2'`?

Reply:
i made this change just now in a follow-on: https://github.com/sul-dlss/dor_indexing_app/pull/1096